src/nile/core/compile.py

```diff
@@ -58,6 +58,9 @@ def _compile_contract(path, directory=None):
         --output {BUILD_DIRECTORY}/{filename}.json \
         --abi {ABIS_DIRECTORY}/{filename}.json
     """
+    if "Account" in str(filename):
+        cmd = cmd + "--account_contract"
```

Comment:
what alternatives do we have? I think this could go wrong easily, like a non-account contract that says account (e.g. "AccountManager" could be a contract that manages accounts). Also we might want to expose this flag somehow with `nile compile`. For Nile's native accounts, we can simply pass the flag internally e.g. in `Account.py`'s constructor.

Reply:
Great point! I was thinking of this as a temp fix so the Cairo Contracts weren't blocked. I'll work on integrating the flag in `nile compile`